Source/Blazorise/Components/Validation/Validation.razor.cs

```diff
@@ -256,23 +249,36 @@ public async Task<ValidationStatus> ValidateAsync( object newValidationValue )
             return await Task.FromResult( Status );
         }
 
+        /// <summary>
+        /// Gets the ValidationHandler for this context.
+        /// </summary>
+        /// <returns></returns>
+        private IValidationHandler GetValidationHandler()
+        {
+            var validationHandlerType = DetermineHandlerType();
+
+            return ( validationHandlerType is not null )
```

Comment:
What if the handler type changes at runtime? This will not work I think in that case.

Reply:
What do you mean? I don't think I did any logical changes to your code. At least not on purpose.

Reply:
I misunderstood part of the code. Sorry about that...

Now I'm looking at your changes but I don't see how it actually cache any handler. Can you explain?

Reply:
> "I'm blind.. already cached...."
So basically I only did a tiny bit of refactoring nothing else.

Reply:
Yeah, that's why I was confused. The linked issue was talking about the caching and nothing was in the PR 😅

Looking good. Merging!